src/Tabs.jsx

```diff
@@ -63,9 +63,15 @@ export class Tabs extends React.Component {
 	}
 }
 Tabs.propTypes = {
-	children: React.PropTypes.arrayOf(
-		React.PropTypes.instanceOf(TabsTab)
-	).isRequired,
+	children: (props, propName, componentName) => {
+		let error;
+		React.Children.forEach(props[propName], child => {
+			if (typeof child === 'undefined' || child.type !== TabsTab) {
```

Comment:
This is good stuff. I can actually use this to require that at least 2 `TabsTab` children are present.